lambdas/choice/src/writer.py

```diff
@@ -1,6 +1,78 @@
 import os
+import boto3
+
+"""
+Processes SNS messages containing user choices by writing choices to Dynamo.
+For event and database record shapes, see https://github.com/sjawhar/enchanted-brain/issues/4.
+"""
+
+
+EVENT_USER_ID_KEY = 'userId'
+EVENT_CHOICE_TYPE_KEY = 'choiceType'
+EVENT_CHOICE_KEY = 'choice'
+EVENT_TIMESTAMP_KEY = 'timestamp'
+
+CHOICE_COLOR = 'CHOICE_COLOR'
+CHOICE_EMOTION = 'CHOICE_EMOTION'
+CHOICE_CHILLS = 'CHOICE_CHILLS'
+CHOICE_IMAGERY = 'CHOICE_IMAGERY'
+
+CONCERT_SONG_ID = os.environ['EVENT_NAME']
+DYNAMODB_TABLE_NAME = os.environ['DYNAMODB_TABLE_NAME']
+
+choice_type_to_choice_key = {
+    CHOICE_COLOR: 'colors',
+    CHOICE_EMOTION: 'emotions',
+    CHOICE_CHILLS: 'chills',
+    CHOICE_IMAGERY: 'imagery'
+}
+choice_types = choice_type_to_choice_key.keys()
+
+dynamodb = boto3.resource(
+    'dynamodb',
+    region_name=os.environ['AWS_DYNAMO_REGION'],
+    endpoint_url=os.environ['AWS_DYNAMO_ENDPOINT'],
```

Comment:
Outside of local development, we're going to want to omit `endpoint_url`. Might want to do a `dynamodb_args` dict, add `endpoint_url` is `os.environ[DYNAMODB_ENDPOINT]` is set, then `boto3.resource('dynamodb', **dynamodb_args)`

Reply:
Good suggestion. Fixed.